tests/src/test/scala/system/packages/CloudantFeedTests.scala

```diff
@@ -573,7 +573,14 @@ class CloudantFeedTests
                     "authKey" -> wskProps.authKey.toJson,
                     "query_params" -> JsObject("type" -> JsString("avocado"))
                 ))
-                feedUpdateResult.stdout should include("ok")
+
+                withActivation(wsk.activation, feedUpdateResult) {
+                    activation =>
+                        activation.response.success shouldBe true
+                }
+
+                println("Giving the trigger service a moment to process the update")
+                Thread.sleep(10000)
```

Comment:
Can we do this in more deterministic way like poolling with a max try and some smaller sleep between tries?

Reply:
Meant deterministic or do pilling with a chance to get the answer faster